src/styles/main.scss

```diff
@@ -1,3 +1,73 @@
+$days: 28;
```

Comment:
How can I make calendar for 31 days? 
Add a modifier month-length for the calendar with values 28, 29, 30 and 31 (use @for)
It sets the last day to show (use [nth-child](https://css-tricks.com/how-nth-child-works/)) 
![2023-01-20_14h41_27](https://user-images.githubusercontent.com/61904995/213697353-a93486de-fead-49ed-ba6d-035497a6055c.png)

Reply:
@IvanFesenko I don't understand what is problem. You can change variable $days and Calendar will rebuilt.

Reply:
@IvanFesenko I don't understand what is problem. You can change variable $days and Calendar will rebuilt.

Reply:
![image](https://user-images.githubusercontent.com/122163374/213701778-4c65f183-5c67-4abd-8dd8-79b53d07e29d.png)

Reply:
didn't change it, if you have questions add them to chat